ehr/src/org/labkey/ehr/utils/TriggerScriptHelper.java

```diff
@@ -1278,6 +1376,16 @@ public void setQuantity(double quantity)
             _quantity = quantity;
         }
 
+        public void setIsInTransaction(boolean inTransaction)
```

Comment:
I understand the new variable, but do you actually need to expose a setter?

Reply:
probably not, this can be removed